jsonnet/kube-thanos/util.libsonnet

```diff
@@ -0,0 +1,8 @@
+{
+  withOptionalArgs(config, flagsToKeysMap)::
+    [
+      '--%s=%s' % [flag, config[flagsToKeysMap[flag]]]
+      for flag in std.objectFields(flagsToKeysMap)
+      if config[flagsToKeysMap[flag]] != null
+    ],
+}
```

Comment:
This should be needed in the end :blush:

Reply:
Will wait for resolution to https://github.com/thanos-io/kube-thanos/pull/187/files#r555236740, and if you still want to remove this function I can do that :+1: